### DISCUSSION TITLE
Welcome to Hedging-of-Financial-Derivatives Discussions!

### DISCUSSION THREAD

**Discussion by Akshat111111** [Akshat111111/Hedging-of-Financial-Derivatives#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by michaelfred533** [comment]
I tested out the code with energy sector stocks in the NYSE and they had great performance, the pair had really close cointegration and preformed much better than the other sectors I tested so far. I can add that script to the project too if you're interested

**Reply by Akshat111111** [reply]
Yeah please add it too.Am really waiting to see it's performance.

**Comment by Amarta113** [comment]
@Akshat111111 Please add labels on the issues

**Reply by Akshat111111** [reply]
Done

**Reply by Amarta113** [reply]
@Akshat111111 Also add Level no.

**Reply by Akshat111111** [reply]
Done

**Comment by Amarta113** [comment]
@Akshat111111 I have submitted PR for issue #59. Since this is my first time working on it, I’m not entirely sure if everything is correct.. Can you please review it

**Reply by Akshat111111** [reply]
yeah sure.

**Comment by shubhamnakum** [comment]
![Screenshot 2024-05-19 202448](https://github.com/Akshat111111/Hedging-of-Financial-Derivatives/assets/122023174/d0dfc29b-2815-4970-b168-669f4956a094)

i am having this issue while cloning. what is this, what should i do? any one have any idea of this?

**Reply by Akshat111111** [reply]
you can retry,If then also same error, so you can use github workspace.

**Reply by shubhamnakum** [reply]
![Screenshot 2024-05-19 203926](https://github.com/Akshat111111/Hedging-of-Financial-Derivatives/assets/122023174/09c0dda6-5943-4fdd-8dd8-362384bdbef9)

i deleted the existing forked repo and then again tried the process but then i got some new issues.

**Reply by Amarta113** [reply]
@shubhamnakum ask chatgpt  or you can clone via vscode. 
Click on the clone git repo and put url of the repository
![Screenshot 2024-05-19 205909](https://github.com/Akshat111111/Hedging-of-Financial-Derivatives/assets/134733248/ed6606a8-86d1-4922-a8ed-9bca6c52f3ad)

 I think this may help you to resolve that error

**Reply by Akshat111111** [reply]
yes, this should work.

**Comment by Amarta113** [comment]
@Akshat111111 I have performed sentiment analysis using the ChatGPT API. Kindly create an issue for this.

**Reply by Akshat111111** [reply]
You can create the issue and explain some of the features and then I will assign you

**Comment by Amarta113** [comment]
@Akshat111111 how to handle this error? I recently generated a new API key
![image](https://github.com/Akshat111111/Hedging-of-Financial-Derivatives/assets/134733248/97132ceb-8b27-4578-acb1-d1109b27540f)
If possible, could you test this code using your own API key?

**Reply by Akshat111111** [reply]
I think the error is due to the api, you can try with some other api key

**Comment by Amarta113** [comment]
@Akshat111111 I have received this notification multiple times. How should I handle it? 
![image](https://github.com/Akshat111111/Hedging-of-Financial-Derivatives/assets/134733248/ae8c11f0-43d1-4d2b-8a8b-34906726e796)

**Reply by Akshat111111** [reply]
Let me sort it out

**Reply by Amarta113** [reply]
@Akshat111111 The problem is not resolved yet. I think , this workflow issue is related to my forked repositories because it originates from those repositories. Can you guide me on how to handle it?

![image](https://github.com/Akshat111111/Hedging-of-Financial-Derivatives/assets/134733248/abe7665f-8ef3-44d9-bbac-2b76c945e54b)

**Reply by Akshat111111** [reply]
These errors are mainly due to the workflow files. You can check the workflow files of your forked copy
